src/main/java/com/greenfoxacademy/goddesstribesbackend/models/entities/Townhall.java

```diff
@@ -0,0 +1,42 @@
+package com.greenfoxacademy.goddesstribesbackend.models.entities;
+
+import javax.persistence.Entity;
+
+@Entity
+public class Townhall extends Building {
+
+  public static int START_FOOD_AMOUNT = 50;
+  public static int START_GOLD_AMOUNT = 100;
+  private static final int START_FOOD_CAPACITY = 1000;
+  private static final int START_GOLD_CAPACITY = 1000;
+
+  private int foodCapacity;
+  private int goldCapacity;
+
+  public Townhall() {
+  }
+
+  public Townhall(Kingdom kingdom) {
+    super(kingdom);
```

Comment:
When do we use kingdom in here? Why does this need the kingdom? Is it needed here? 
Maybe we want to extend a repository where we do the pre-filtering in a generic way? 
I am not sure yet what's the easiest way.
Let's leave this like this for now, but we will talk about why this or that is any better on Wednesday.
(Remind me if I'd forget)